src/GitTools.Core/GitTools.Core.NET40/Git/AuthenticationInfo.cs

```diff
@@ -0,0 +1,8 @@
+namespace GitTools.Git
+{
+    public class AuthenticationInfo
```

Comment:
I haven't looked into the auth options with libgit2sharp, but we will need to look at that then support what they have